src/api.ts

```diff
@@ -2,28 +2,27 @@ import { Hono } from "hono";
 import _ from "lodash";
 import { Field } from "./game";
 
-export const api = new Hono();
+const app = new Hono()
```

Comment:
let's keep this as `api`, not to be confused with the `app` in `src/index.tsx`, and it can still be a non-default export, your conventions have grown on me 😅
```suggestion
export const api = new Hono()
```